activerecord/lib/rails/generators/active_record/model/templates/model.rb.tt

```diff
@@ -9,5 +9,8 @@ class <%= class_name %> < <%= parent_class_name.classify %>
 <% if attributes.any?(&:password_digest?) -%>
   has_secure_password
 <% end -%>
+<% attributes.select { |a| !a.reference? && a.required? }.each do |attribute| -%>
+  validates :<%= attribute.name %>, presence: true
```

Comment:
How about a single `validates_presence_of` line?

Reply:
@kaspth Thanks for the comment. `Rails/Validation` cop in rubocop-rails recommends `Use validates :attribute, hash of validations` by default. So I think it's good to use `validates :attribute` (However, this is not strong opinion. I think it's good to use `validates_presence_of` as well.)

https://github.com/rubocop-hq/rubocop-rails/blob/master/config/default.yml#L435-L441